docs/source/ci.rst

```diff
@@ -76,3 +76,35 @@ Specifically for mongo, the :code:`- image: <SERVICE IMAGE>` portion should look
       command: "mongod --journal"
 
 You will receive a `ContainerCheckFailed: Unable to connect to [...] Mongo test container` error in CI if the above is not added to you job config.
+
+
+GitLab
+------
+For :code:`pytest-mock-resources` to work on GitLab use of :code:`dind` service is required.
+Below is a sample configuration:
+
+.. code-block:: yaml
+
+    services:
+      - docker:dind
```

Comment:
I'm surprised https://docs.gitlab.com/ee/ci/services/postgres.html#use-postgresql-with-the-docker-executor, normal service starting doesnt work.

Can tests not access other services?

Reply:
They can. However, you won't to use the same configuration on CI as you would on your local machine.

Reply:
Aah, okay.

Ultimately I think I would prefer this to more closely mimic the circleci section by making direct use of its service features (and we could include a separate section on using with `dind` in general).

But I'm very happy to accept this as-is, as we have no documentation about it, and I'm not familiar enough with it right now to come up with more detailed docs!

Thanks!

Reply:
Hmm. What do you mean 'service', pytest-mock-resources can make use of an existing docker container?